.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+- name: 🙋🏾🙋🏼‍Question
+  url: https://github.com/layer5io/meshery/discussions/new
+  about: Submit your question using GitHub Discussions.
```

Comment:
Should this be kept? as the discussion is not enabled in this repo, and this will point to the meshery discussion

Reply:
@aneeshnema, good call. Yes, centralizing discussions makes sense. 👍